.github/workflows/docker.yml

```
@@ -81,15 +82,13 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
-      - uses: actions/setup-python@v6
+      - uses: astral-sh/setup-uv@v7
         with:
           python-version: "3.10"
-
-      - name: Create Python 3 virtualenv
-        run: python3.10 -m venv .venv
+          enable-cache: false
 
       - name: Install kraken-wrapper
-        run: pip install uv==0.3.0 && uv tool install kraken-wrapper==0.39.0
+        run: uv tool install kraken-wrapper==0.39.0
```

let's update to the latest kraken-wrapper

```suggestion
        run: uv tool install kraken-wrapper==0.48.4
```